src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -183,95 +183,72 @@ describe('test conversion of DatasetAccessRequest', () => {
 });
 
 describe('test HighlightConceptName', () => {
+  const createHighlightConceptName = (beforeHighlight: string, highlightWord: string, afterHighlight: string) => {
+    return div({ style: { display: 'flex' } }, [
```

Comment:
Do we need to duplicate this here? Can you call either an abstracted method or just call HighlightConceptName?